whats_fresh/whats_fresh_api/views/entry/vendors.py

```diff
@@ -139,14 +140,14 @@ def vendor(request, id=None):
                     'product': vendor_product.product_preparation.product.name
                 })
     elif request.method != 'POST':
-        title = "Add a Vendor"
+        title = "New " + get_fieldname.get_fieldname('vendors')
```

Comment:
@tschuy I think we should keep this as `New Items` instead of `New 'form label'` in all the forms, I think it would be better, what say?